tests/browsing_context/test_navigate_events.py

```diff
@@ -69,7 +71,60 @@ async def test_navigate_checkEvents(websocket, context_id, url_base,
             }
         })
 
-    messages = await read_messages(6,
+    messages = await read_messages(7,
+                                   keys_to_stabilize=KEYS_TO_STABILIZE,
+                                   check_no_other_messages=True,
+                                   sort=False)
+    assert messages == snapshot(exclude=SNAPSHOT_EXCLUDE)
+
+
+@pytest.mark.asyncio
+async def test_window_open_url_checkEvents(websocket, context_id, url_example,
+                                           read_messages, snapshot):
+    await subscribe(websocket, ["browsingContext"])
+
+    # Make same-document navigation.
```

Comment:
this and the next test do not appear to be testing same document navs?

Reply:
correct